cromwell_tools/cromwell_tools.py

```diff
@@ -1,34 +1,144 @@
 """This module contains utility functions to interact with Cromwell.
 """
-import requests
-from requests.auth import HTTPBasicAuth
 import io
-import requests
 import zipfile
+import json
 from StringIO import StringIO
+from datetime import datetime, timedelta
+import time
+import requests
+from requests.auth import HTTPBasicAuth
 
 
-def start_workflow(wdl_file, zip_file, inputs_file, inputs_file2, options_file, url, user=None, password=None):
+_failed_statuses = ['Failed', 'Aborted', 'Aborting']
+
+
+def harmonize_credentials(secrets_file=None, cromwell_username=None, cromwell_password=None):
```

Comment:
I don't think we should provide the option of passing the username and password on the command line, because it's a security risk: https://serverfault.com/questions/592744/how-to-hide-a-password-passed-as-command-line-argument

Reply:
Huh ok, I copied this over as-was in secondary-analysis. But, good point, and we can address this.